src/Frontend/Modules/FormBuilder/Widgets/Form.php

```diff
@@ -518,8 +520,10 @@ private function validateForm()
                 $redirect .= (stripos($redirect, '?') === false) ? '?' : '&';
                 $redirect .= 'identifier=' . $this->item['identifier'];
 
-                // redirect with identifier
-                \SpoonHTTP::redirect($redirect);
+                throw new RedirectException(
```

Comment:
Is there a reason for ?
I think it can be handy to also have a redirect function available in the widgets